test/bats.bats

```diff
@@ -191,6 +191,105 @@ fixtures bats
   [ "${lines[2]}" = "# (in test file $FIXTURE_ROOT/failing.bats, line 4)" ]
 }
 
+@test "find_library_load_path finds single-file libraries with the suffix .bash" {
```

Comment:
I think with the amount of tests here, an own file for test of load and affiliated functions would be in order.